.github/workflows/anchore.yml

```diff
@@ -1,28 +1,19 @@
-# This workflow uses actions that are not certified by GitHub.
-# They are provided by a third-party and are governed by
-# separate terms of service, privacy policy, and support
-# documentation.
-
-# This workflow checks out code, builds an image, performs a container image
-# vulnerability scan with Anchore's Grype tool, and integrates the results with GitHub Advanced Security
-# code scanning feature.  For more information on the Anchore scan action usage
-# and parameters, see https://github.com/anchore/scan-action. For more
-# information on Anchore's container image scanning tool Grype, see
-# https://github.com/anchore/grype
-name: Anchore Container Scan
+name: Vulnerability Scan
 
 on:
-  push:
-    branches: [ "main" ]
+  release:
```

Comment:
@pratikjagrut What's the reason behind this change? I'm only concerned if we do this on release, we don't run it often. We only know if it fails / succeeds only on releasing, which could be a surprising failure.

Reply:
Oh I realize there's a PR too, so never mind